README.md

```diff
@@ -1,6 +1,33 @@
 # BMX
 
-BMX grants you API access to your AWS accounts, based on Okta credentials that you already own.
-It uses your Okta identity to create short-term AWS STS tokens, as an alternative to long-term IAM access keys.
+BMX provides API access to your AWS accounts using existing Okta credentials. Instead of long-term IAM access keys, BMX creates short-term AWS STS tokens using your Okta identity.
 
-This is a work in progress dotnet 7 branch
+# Installation
+
+Download the appropriate binary from the [releases](https://github.com/Brightspace/bmx/releases) page. Place the executable in a convenient location within your file system.
+
+# Commands
+
+## configure
+
+The `bmx configure` command creates or updates the global BMX configuration file, located at `~/.bmx/config`. This file can store your Okta organization, username, and your AWS default session duration. Okta account sessions are saved when a configuration file is present. For security reasons, it's recommended to avoid running `bmx configure` or creating a global configuration file on a machine used by multiple users.
```

Comment:
Felt like configure needed a bit more than 1 sentence just for the fact about multi user machine and the worries we had with that

Reply:
But do we trust someone to read it if they are only linked directly to the configure section

Reply:
We have this now:
https://github.com/Brightspace/bmx/blob/4094b59e90dc810dfc1193986faaf409108f9ad5/src/D2L.Bmx/OktaAuthenticator.cs#L76-L77
which I think adequately addresses possible user confusion.